pkg/features/kube_features.go

```diff
@@ -845,7 +845,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	CSIMigrationRBD: {Default: false, PreRelease: featuregate.Alpha}, // Off by default (requires RBD CSI driver)
 
-	CSIMigrationvSphere: {Default: true, PreRelease: featuregate.Beta},
+	CSIMigrationvSphere: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.26 (requires vSphere CSI Driver)
```

Comment:
There should be only comment `// remove in 1.28`, so we know when to remove the flag.

Reply:
done.